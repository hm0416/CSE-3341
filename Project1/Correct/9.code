program
	ref x;
begin
	x = new;
	x = 1;
	if x<100 then
		ref x;
		x = new;
		x = 10;
		output x;
	endif
	output x;
end