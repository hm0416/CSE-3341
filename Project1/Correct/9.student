INT
COMMA
COMMA
COMMA
COMMA
SEMICOLON
SEMICOLON

