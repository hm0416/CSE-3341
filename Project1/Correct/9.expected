gc:1
gc:2
10
gc:1
1
gc:0
